hatch_build.py

```
@@ -43,5 +45,16 @@ def initialize(self, version, build_data):
+            "python",
+            "-m",
+            "sass_embedded.dart_sass",
+            "--clean",
+            "--os",
+            release["os"],
+            "--arch",
+            release["arch"],
+        ]
+        if release["is_musl"]:
+            cmd.append("--musl")
+        subprocess.run(cmd, cwd=here / "src")
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)